torch/onnx/symbolic_opset12.py

```diff
@@ -13,3 +13,7 @@
 def einsum(g, equation, tensor_list):
     tensors = sym_help._unpack_list(tensor_list)
     return g.op("Einsum", *tensors, equation_s=equation)
+
+@parse_args('s', 'v', 'v')
+def mse(g, reduction, input, target):
```

Comment:
the function name should be mse_loss not mse..
I tried running the tests that you added and saw that there was the op 'aten::broadcast_tensors' in the jit trace but we don't have export support for it.

Reply:
Even if the tests are disable, run them locally and verify that the graph exports correctly